scripts/drop-oracle-tables.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
Use `set -ex pipefail` so it exits early on any subprocess that has non-zero exit code